src/zoo.js

```diff
@@ -8,59 +8,104 @@ eslint no-unused-vars: [
   }
 ]
 */
-
 const data = require('./data');
 
-function animalsByIds(ids) {
-  // seu código aqui
+function animalsByIds(...ids) {
+  if (!ids) return [];
+  return ids.map(elemento => data.animals.find(arr => (arr.id === elemento)));
 }
 
-function animalsOlderThan(animal, age) {
-  // seu código aqui
+function animalsOlderThan(animal, idade) {
+  const filterFunction = data.animals.find(arr => arr.name === animal);
```

Comment:
Object destructuring

```suggestion
function animalsOlderThan(animal, idade) {
  const filterFunction = data.animals.find(({ name }) => name === animal);
```